src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php

```diff
@@ -304,6 +304,27 @@ class_exists('Test\\'.__NAMESPACE__.'\\UseTraitWithInternalMethod', true);
 
         $this->assertSame(array(), $deprecations);
     }
+
+    public function testVirtualUse()
+    {
+        $deprecations = array();
+        set_error_handler(function ($type, $msg) use (&$deprecations) { $deprecations[] = $msg; });
+        $e = error_reporting(E_USER_DEPRECATED);
+
+        class_exists('Test\\'.__NAMESPACE__.'\\ExtendsVirtual', true);
+
+        error_reporting($e);
+        restore_error_handler();
+
+        var_dump($deprecations);
```

Comment:
can you write the actual assertion instead ?

And we also need a test where the class is implementing the method defined virtually in the interface, to ensure that we don't get a deprecation for such case (as we have the method)

Reply:
of course, meanwhile do you still have fixture suggestions?

Reply:
> And we also need a test where the class is implementing the method defined virtually in the interface

Done. Ill update fixtures once the implementation is OK.